mne/io/eeglab/eeglab.py

```diff
@@ -60,8 +59,7 @@ def _to_loc(ll):
 
 
 def _get_info(eeg, montage, eog=()):
```

Comment:
should channel information be added via the set_montage. I would suggest that there would be a routine that can extract the channel information from the either the `set` or the `locs` files, and create a montage object. that montage would then be applied using `set_montage`. It is the cleaner approach.